tools/brownie-example/README.md

```diff
@@ -0,0 +1,219 @@
+
+   Example:
+   ```python
+   tx = contract.someFunction({'from': accounts[0]})
+   tx.info()  # View transaction details
+   ```
+
+2. **Accessing Events**:
+   - Events emitted by contracts during a transaction can be easily accessed through the transaction object. You can retrieve all events or filter for specific ones.
+
+   Example:
+   ```python
+   tx = contract.someFunction({'from': accounts[0]})
+   print(tx.events)  # Print all emitted events
+   ```
+
+   To access a specific event, you can reference it by name:
+   ```python
+   transfer_event = tx.events['Transfer']
+   print(transfer_event)
+   ```
+
+3. **Real-Time Event Monitoring**:
+   - You can set up filters to watch for specific events in real time. This is useful when you want to monitor certain actions as they happen on the network.
+
+   Example:
+   ```python
+   event_filter = contract.events.Transfer.createFilter(fromBlock='latest')
+   events = event_filter.get_all_entries()
+   print(events)
+   ```
+
+4. **Reverting and Snapshotting**:
+   - When testing on forked networks, you can create a snapshot of the current blockchain state, perform tests, and then revert back to the snapshot for repeated testing.
+
+   Example:
+   ```python
+   network.snapshot()
+   tx = contract.someFunction({'from': accounts[0]})
+   network.revert()  # Revert to the snapshot
+   ```
+
+   You can configure Brownie to automatically isolate each test by adding the following code to your conftest.py file:
+   ```python
+   @pytest.fixture(scope="function", autouse=True)
+   def isolate(fn_isolation):
+     pass
+   ```
+   This ensures that each test runs in an isolated environment, preventing any side effects from one test affecting another.
+
+5. **Revert Reasons**:
+   - If a transaction fails or reverts, Brownie provides detailed error messages and stack traces, including revert reasons, making it easier to identify and fix issues in your contract logic.
+
+# Brownie usage example
+
+Simple scripts for basic operations like hbars transfer, balance fetching, and contract interactions (deployment and calls).
+
+## Setup & Install
+
+Follow this instruction from the installation section above:
+
+```Shell
+pip install eth-brownie
+```
+
+## Running tests
+
+After installing brownie you can go to your project directory and run following script:
+
+1. Run a fork of the network of your choose (snapshot method is required for tests).
+
+   In order to run the tests on the non-forked (no snapshot method available) network 
+   you need to have 2 accounts configured in brownie
+   
+   Bob's account can not be empty:
+   
+   ```shell
+   brownie accounts new bob
+   brownie accounts new alice
+   ```
+   Since the transfer of hbars is performed during the tests it is adviced to connect to the forked network instead of
```

Comment:
Spelling.  It should read "it is advised..."

Reply:
Yes, you are right. Thank you for noticing it.